gulp/tasks/templates.js

```diff
@@ -1,120 +1,157 @@
-var _ = require('lodash'); // to handle collections gracefully
-var config = require('../config').templates; // pull in the pathing config file
-var fs = require('fs'); // used to work with substack's module
-var glob = require('glob'); // to dynamically read in all content md files
-var gulp = require('gulp'); // because this is a gulp task. duh.
-var md = require('markdown-it')({ html: true }); // to convert markdown to html
-var source = require('vinyl-source-stream'); // used to convert substack's readStream to vinylStream
-var moment = require('moment-timezone');
-var exec = require('child_process').exec
-var utils = require('../util/template-utils.js');
+/*
+ * 1. because this is a gulp task. duh.
```

Comment:
Haha, I only left this in because the original version has that exact comment ;)  I felt bad deleting too much..

Fair enough though, I can take out the more obvious ones (I actually removed comments re: `fs` and `path` prior to committing as they seemed, well, obvious.)